src/odemis/gui/cont/menu.py

```diff
@@ -149,6 +149,15 @@ def __init__(self, main_data, main_frame):
         # /Help/About
         main_frame.Bind(wx.EVT_MENU, self._on_about, id=main_frame.menu_item_about.GetId())
 
+        # add a toggle for correlation tab in viewer mode
+        if main_data.is_viewer:
+            main_frame.Bind(wx.EVT_MENU, self._on_show_correlation, id=main_frame.menu_item_show_correlation.GetId())
+        else:
+            # remove the menu item
+            menu = main_frame.menu_item_show_correlation.GetMenu()
+            menu.Remove(main_frame.menu_item_show_correlation)
+            main_frame.menu_item_show_correlation.Destroy()
+
     def _on_update(self, evt):
         import odemis.gui.util.updater as updater
```

Comment:
Not a part of the review, but is it fine to have import statement in between the file?

Reply:
It depends. Usually, you want to have all imports at the top of the file. However, if you want to only import "dynamically", just before a function is needed, then it's useful. I think that here the reasoning is that this function is only useful on Windows, so no need to import the module for the most common use cases (on Linux). IIRC, a long time ago, the import would even fail on Linux, hence it was placed here. Nowadays, that import should work on Linux, and be essentially a no-op, as long as you don't call any functions, so we could move it back to the top. (but it's fine here)

Reply:
Thank you Eric!